capture.sh

```diff
@@ -2,6 +2,15 @@
 
 SCREENCAPTURE_CMD='unknown'
 
+# Parse the command line arguments
+while [ $# -gt 0 ]; do
+    if [[ $1 == *"--"* ]]; then
+      v="${1/--/}"
+      declare $v="$2"
+    fi
+  shift
+done
```

Comment:
Sorry for being pedantic, but I think that `if` and `shift` should be at the same level:

```suggestion
# Parse the command line arguments
while [ $# -gt 0 ]; do
  if [[ $1 == *"--"* ]]; then
    v="${1/--/}"
    declare $v="$2"
  fi
  shift
done
```